pkg/network/ebpf/c/protocols/http2/decoding-defs.h

```diff
@@ -18,9 +18,12 @@
 // Currently we have up to 120 frames in a packet, thus 7 (7*18 = 126) tail calls is enough.
 #define HTTP2_MAX_TAIL_CALLS_FOR_HEADERS_PARSER 7
 #define HTTP2_MAX_FRAMES_FOR_HEADERS_PARSER (HTTP2_MAX_FRAMES_FOR_HEADERS_PARSER_PER_TAIL_CALL * HTTP2_MAX_TAIL_CALLS_FOR_HEADERS_PARSER)
-// Maximum number of frames to be processed in a single TCP packet. That's also the number of tail calls we'll have.
-// NOTE: we may need to revisit this const if we need to capture more connections.
+// Maximum number of frames to be processed in a single TCP packet.
 #define HTTP2_MAX_FRAMES_ITERATIONS 120
+// This represents a limit on the number of tail calls that can be executed within the frames_filter program.
+// The number of frames to parse is determined by HTTP2_MAX_FRAMES_ITERATIONS, resulting in a total defined as:
+// HTTP2_MAX_FRAMES_ITERATIONS * HTTP2_MAX_TAIL_CALLS_FOR_FRAMES_FILTER (for tail call 0-1)
+#define HTTP2_MAX_TAIL_CALLS_FOR_FRAMES_FILTER 1
```

Comment:
should be 2

Reply:
So this is not so accurate, We will start with the frame filter for the first time, then I have added the option to use one more tail call (we want to reach 240, so we need only 1 tail call )

Reply:
it still should be 2
you're starting to count from 0, which makes it unreadable

Reply:
https://github.com/DataDog/datadog-agent/pull/21755#discussion_r1439238581